crates/control/src/behavior/calibrate.rs

```diff
@@ -0,0 +1,11 @@
+use types::{MotionCommand, PrimaryState, WorldState};
+
+pub fn execute(world_state: &WorldState) -> Option<MotionCommand> {
+    match world_state.robot.primary_state {
+        PrimaryState::Calibration => Some(MotionCommand::Stand {
+            head: types::HeadMotion::Unstiff,
```

Comment:
Warning, this currently does not actually unstiff the head.
The head is always driven with 0.8 stiffness as implemented [here](https://github.com/HULKs/hulk/blob/main/crates/control/src/motion/head_motion.rs#L90-L96).

This does allow one to move the head around by hand because the unstiff motion always requests the currently measured joint angles and thus is subject to induced drift. However, the head angles are still clamped, for example negative pitch values are not allowed.
If that's fine for you, I'm fine with it, but I also tried it 0 stiffness which makes it much easier to move the head by hand but still keeps its position due to the weird motor brake "feature" and also allows a larger range of head positions.

Reply:
Wtf, unstiff is not unstiff... :exploding_head: I will correct this.

Reply:
Done